__tests__/app.test.js

```diff
@@ -23,4 +23,13 @@ describe('/api/topics', () => {
                 
             })
     })
+    describe('/api', () => {
+        test('200: returns all available endpoints and descriptions', () => {
+            return request(app).get('/api')
+                .expect(200)
+                .then((response) => {
+                expect(Object.keys(response.body)).toEqual([ 'GET /api', 'GET /api/topics', 'GET /api/articles' ])
+            })
```

Comment:
You will need to update this test every time you update your endpoints.json, if you don't want to do that you can just require in the json file and check that the response looks the same as it?